### DISCUSSION TITLE
【 一起共建吧】您最期待XIAOJUSURVEY开源什么内容？

### DISCUSSION THREAD

**Discussion by sudoooooo** [didi/xiaoju-survey#11] [Q&A]
已开源内容[可查](https://xiaojusurvey.didi.cn/docs/document/%E4%BA%A7%E5%93%81%E6%89%8B%E5%86%8C/%E6%A6%82%E8%BF%B0)，建设中内容[可查](https://github.com/didi/xiaoju-survey/issues/45)，一起来聊聊您最期待的开源内容吧！

描述您的场景和期待的能力～
<img src="https://img-hxy021.didistatic.com/static/starimg/img/yUahvd3x9d1701172623531.png"  width="150" />

**Comment by aairyy** [comment]
👍 界面皮肤可配置的话就更好啦🙋比如主题色、按钮样式这种，这样落地起来更贴合不同品牌的视觉～

**Comment by youuss** [comment]
发现问卷web是建立在`element-plus`基础上，那后续有规划建立属于小桔问卷的基础组件库吗？然后再在基础组件库上去做物料的拓展🤔

**Reply by sudoooooo** [reply]
你好，感谢交流。

物料市场确实在我们建设计划当中，会综合考虑基础库的选择，有具体Roadmap后会及时同步给社区🤝。

现阶段更聚焦在打造调研基座、扩展技术栈和智能化建设：欢迎了解[生态理念和发展计划](https://xiaojusurvey.didi.cn/docs/next/community/%E7%94%9F%E6%80%81%E5%BB%BA%E8%AE%BE)
